packages/core/src/main/conversation/ConversationService/ConversationService.ts

```diff
@@ -144,20 +144,17 @@ export class ConversationService {
     return genericMessage;
   }
 
-  private async getConversationQualifiedMembers(conversationId: QualifiedId): Promise<QualifiedId[]> {
+  private async getConversationQualifiedMembers(conversationId: string | QualifiedId): Promise<QualifiedId[]> {
     const conversation = await this.apiClient.api.conversation.getConversation(conversationId);
     /*
      * If you are sending a message to a conversation, you have to include
      * yourself in the list of users if you want to sync a message also to your
      * other clients.
      */
-    return (
-      conversation.members.others
-        .filter(member => !!member.qualified_id)
-        .map(member => member.qualified_id!)
-        // TODO(Federation): Use 'domain' from 'conversation.members.self' when backend has it implemented
-        .concat({domain: this.apiClient.context!.domain!, id: conversation.members.self.id})
-    );
+    return conversation.members.others
+      .filter(member => !!member.qualified_id)
+      .map(member => member.qualified_id!)
+      .concat(conversation.members.self.qualified_id!);
```

Comment:
Are we absolutely sure backend now implements this everywhere?

Reply:
I tested on staging.zinfra.io and it works